network/src/protocols/wire/handshake/v1/mod.rs

```diff
@@ -2,13 +2,15 @@
 // SPDX-License-Identifier: Apache-2.0
 
 //! This module defines the structs transported during the network handshake protocol v1.
-//! These should serialize as per [link](TODO: Add ref).
+//! These should serialize as per the [DiemNet Handshake v1 Specification].
 //!
 //! During the v1 Handshake protocol, both end-points of a connection send a serialized and
-//! length-prefixed `HandshakeMsg` to each other. The handshake message contains a map from
+//! length-prefixed [`HandshakeMsg`] to each other. The handshake message contains a map from
```

Comment:
What's the purpose of the brackets btw?  Will it link back to the original type?

Reply:
Right, it'll link to the definition in the rendered rustdocs. You can view the rendered version with:

```
cargo doc --all-features -p network --open
```